velox/functions/lib/string/StringImpl.h

```diff
@@ -472,13 +472,20 @@ FOLLY_ALWAYS_INLINE bool splitPart(
 template <
     bool leftTrim,
     bool rightTrim,
-    typename TShouldTrim,
     typename TOutString,
-    typename TInString>
+    typename TInString,
+    typename TShouldTrim>
```

Comment:
Why change the order of template argument?

Reply:
align order with arguments and make method call simpler